README.md

```diff
@@ -17,11 +17,8 @@ func main() {
 	// * Ex. (func (export "fac") (param i64) (result i64) ...
 	source, _ := os.ReadFile("./tests/engine/testdata/fac.wasm")
 
-	// Decode the binary as WebAssembly module.
```

Comment:
there was no use of this type except passing it to instantiate, so merging it with config is a net win for simplicity. If we ever want to make a wat2wasm thing we can do that as a separate utility and without messing up this api